cypress/integration/onboarding/messaging/whitespace_does_not_affect_emoji_size_spec.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 5]*/
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    // Unpin all posts at the end of the test
+    after(() => {
+        // pinnedPosts.forEach((pinnedPost) => {
+        //     cy.apiUnpinPosts(pinnedPost);
+        // });
+    });
```

Comment:
Maybe missed to remove after copy/paste?

Reply:
Removed